iset.mm

```diff
@@ -86379,15 +86532,35 @@ finite and infinite sets (and therefore if the set size is a nonnegative
 $)
 
   $c NN0* $. $( The set of extended nonnegative integers $)
+  $c NN+oo $. $( Set of descending sequences in ` 2o ^m _om ` $)
```

Comment:
descending --> non-increasing

But wouldn't it be more natural to take non-decreasing instead ? Then, zero maps to the zero sequence. I don't remember what Escardo does.